csrc/host_ir/executor.cpp

```diff
@@ -281,6 +281,25 @@ void HostIrExecutor::handle(EndCoalescing* end_coalescing) {
   works_[end_coalescing] = backend->endCoalescing();
 }
 
+void HostIrExecutor::handle(MatmulOp* matmul) {
```

Comment:
I recommend we move this and the unit test in test_host_irs.cpp to a different PR. https://google.github.io/eng-practices/review/developer/small-cls.html

Reply:
The present PR is stacked on top of https://github.com/NVIDIA/Fuser/pull/3171 which does what you suggest